### PR TITLE
DOCSP-23597 Remove limitations

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-23597-remove-limitations-v1.0/reference/limitations/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-23597)

### CODE REVIEW

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -65,9 +64,7 @@ discuss your requirements.
 Unsupported Collection Types
```
Comment (jeff-allen-mongo) [inline source/reference/limitations.txt:64]: Now that there is only one unsupported collection type, consider making this heading more specific:

```suggestion
Time Series Collections
```

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM with a suggestion

File: source/reference/limitations.txt (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/reference/limitations.txt:64]: Actually I see in PR #51 that we're adding another bullet to this section, so this can stay as-is.

**Comment by davemungo** [comment]
@ldennis  would you mind giving this one review since Alan is out?

**Review by ldennis — APPROVED** [review]
LGTM

**Pull request merged**